packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -14,19 +15,26 @@ enum IndexChange {
 interface MenuProps {
   readonly visible: boolean;
   readonly options: Option[];
+  readonly menuSize: string | undefined;
```

Comment:
I think `size` would be more conventional base on the props on our other components.